Makefile

```diff
@@ -19,6 +19,7 @@ build:
 	docker-compose build
 
 pip-compile:
+	pip install -q pip-tools
```

Comment:
Why do we make pip-compile a prerequisite to pip-install-local in the first place?  If we just had pip-install-local by itself as the first step then we wouldn't need this here as well as in requirements.txt.  I'm not an python dependency management expert though so maybe there's a good reason for this

Reply:
I'm guessing that prerequisite was intended as a convenience to ensure `requirements.txt` is up to date before installing, in case someone had changed `requirements.in` but not run `pip-compile` yet.

Personally I'd be fine with removing that prerequisite (though adding `pip-tools`as a requirement would still be necessary for `pip-compile` to be available).

Reply:
This shouldn't be necessary, if we already add `pip-tools` to requirements would it be a fair assumption here that someone already set up their environment and installed all dependencies?

Reply:
I just noticed the readme just says `Add new Python dependencies into requirements.in and execute make pip-install-local` so I guess it is for convenience.  I would opt to have separate commands for compile and install for transparency but it doesn't really matter.  If we're sure we can remove it then let's do that, otherwise I don't think it matters enough to really dig into